GTMAppAuthSwift/Sources/GTMAppAuthFetcherAuthorization.swift

```diff
@@ -28,9 +28,10 @@ import GTMSessionFetcher
 /// An implementation of the `GTMFetcherAuthorizationProtocol` protocol for the AppAuth library.
 ///
 /// Enables you to use AppAuth with the GTM Session Fetcher library.
-@objc open class GTMAppAuthFetcherAuthorization: NSObject,
-                                                 GTMFetcherAuthorizationProtocol,
-                                                 NSSecureCoding {
+@objc(GTMAppAuthFetcherAuthorization)
```

Comment:
What name is exposed in Objc without this argument?

Reply:
Good question -- it's not what we have here! I think maybe it is module name + . + class name == `GTMAppAuthSwift.GTMAppAuthFetcherAuthorization`.

I used the below* to print out the string name of a Swift class with and without a name within `@objc`. Without a name in there, the fully qualified name was `ModuleName.Foo`. With a name like `@objc(Foo)`, the fully qualified name was `Foo`.

* https://forums.swift.org/t/getting-the-more-fully-qualified-name-of-a-type/14375

Reply:
I don't think you could have a `.` in an Objc class name though, right?

Separately, should we consider merging this to `master`?

Reply:
I'm not sure what you mean by your first question. We don't have a `.` in either name in ObjC or Swift. The fully qualified name is built by THE SYSTEM ™️ and used by `NSKeyed(Un)Archiver`. 

For the second question, we could merge this to master. I'm guessing that it's failing there as well?